client/js/lounge.js

```diff
@@ -750,6 +763,18 @@ $(function() {
 		case "user":
 			$(".channel.active .users .user[data-name=" + $(this).data("data") + "]").click();
 			break;
+		case "user-ban":
+			socket.emit("input", {
+				target: chat.data("id"),
+				text: "/mode +b " + $(this).data("data")
+			});
+		// eslint-disable-next-line no-fallthrough
```

Comment:
Note: I had him do this is to avoid code duplication, as we want to kickban on ban, so we let it fall through. Ban without kick is a valid operation, but is counter to what most people would expect. Might want to rename the menu entry to reflect that if others thinks it's needed.